src/DGtal/topology/CubicalComplexFunctions.ih

```diff
@@ -239,10 +240,10 @@ collapse( CubicalComplex< TKSpace, TCellContainer > & K,
     {
       CellMapIterator cmIt  = *it;
       uint32_t& cur_data    = cmIt->second.data;
-      if ( cur_data & CC::REMOVED ) K.eraseCell( cmIt ); 
+      if ( cur_data & CC::REMOVED ) K.eraseCell( cmIt );
       else                          cur_data &= ~CC::COLLAPSIBLE;
-      // if ( (*it)->second.data & CC::REMOVED ) 
-      //   K.eraseCell( *it ); 
+      // if ( (*it)->second.data & CC::REMOVED )
```

Comment:
cleanup ?

Reply:
CubicalComplex was developed by @JacquesOlivierLachaud. 
See: 2015-11-24 70bbd93257fab56771a3cef9d5b037501f8585a7 Create CubicalComplex functions, operators within, and move collapse as a function. (Jacques-Olivier Lachaud)

I only added a few methods on it.

But I can definitely clean it, but not sure if that commented code is there for future polish.

Reply:
ok. I'll check with @JacquesOlivierLachaud, thx